source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -104,6 +129,232 @@ TEST_F(NiRFmxWLANDriverApiTests, InitializeFromNIRFSA_Close_Succeeds)
   ni::tests::system::EXPECT_SUCCESS(close_response);
 }
 
+TEST_F(NiRFmxWLANDriverApiTests, OFDMModAccTXPCompositeFromExample_FetchData_DataLooksReasonable)
+{
+  auto session = init_session(stub(), PXI_5663E);
+  EXPECT_SUCCESS(session, client::cfg_frequency_reference(stub(), session, "", FREQUENCY_REFERENCE_SOURCE_ONBOARD_CLOCK, 10e6));
+  EXPECT_SUCCESS(session, client::cfg_frequency(stub(), session, "", 2.412e9));
+  EXPECT_SUCCESS(session, client::cfg_reference_level(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_external_attenuation(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_iq_power_edge_trigger(stub(), session, "", "0", IQ_POWER_EDGE_TRIGGER_SLOPE_RISING_SLOPE, -20.0, 0.0, TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO, 5.0e-6, IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE, BOOLEAN_TRUE));
+  EXPECT_SUCCESS(session, client::cfg_standard(stub(), session, "", STANDARD_802_11_AG));
+  EXPECT_SUCCESS(session, client::cfg_channel_bandwidth(stub(), session, "", 20e6));
+  EXPECT_SUCCESS(session, client::select_measurements(stub(), session, "", MEASUREMENT_TYPES_TXP|MEASUREMENT_TYPES_OFDMMODACC, true));
+  EXPECT_SUCCESS(session, client::ofdm_mod_acc_cfg_measurement_length(stub(), session, "", 0, 16));
+  EXPECT_SUCCESS(session, client::ofdm_mod_acc_cfg_averaging(stub(), session, "", OFDM_MOD_ACC_AVERAGING_ENABLED_FALSE, 10));
+  EXPECT_SUCCESS(session, client::txp_cfg_averaging(stub(), session, "", TXP_AVERAGING_ENABLED_FALSE, 10));
+  EXPECT_SUCCESS(session, client::txp_cfg_maximum_measurement_interval(stub(), session, "", 1e-3));
+  EXPECT_SUCCESS(session, client::initiate(stub(), session, "", ""));
+
+  OFDMModAccFetchCompositeRMSEVMResponse ofdm_mod_acc_fetch_composite_rmsevm_response;
+  TXPFetchMeasurementResponse txp_fetch_measurement_response;
+  ofdm_mod_acc_fetch_composite_rmsevm_response = client::ofdm_mod_acc_fetch_composite_rmsevm(stub(), session, "", 10.0);
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_composite_rmsevm_response);
+  txp_fetch_measurement_response = client::txp_fetch_measurement(stub(), session, "", 10.0);
+  EXPECT_WARNING(txp_fetch_measurement_response, kRisingEdgeDetectionFailedWarning);
+  
+  EXPECT_LT(0.0, ofdm_mod_acc_fetch_composite_rmsevm_response.composite_rmsevm_mean());
```

Comment:
Some of these field names will be broken by David's change right? Or is wlan not included in that yet?

Reply:
Yeah, you're right. I think his is getting close so I'll wait for #499 to go in and then merge main and fix up the names.

Reply:
I can also regen after the update, so Ryan can merge that in rather than doing it manually.

Reply:
Done. You can find the updates at the usual place, @reckenro -- also rebased on top of main.

Reply:
Updated from your branch. Got the updates for the acronym names and the field names on one line.